src/components/ec-btn/types.ts

```diff
@@ -0,0 +1,32 @@
+import type { RouteLocationRaw } from 'vue-router';
+
+import type { IconName } from '../ec-icon/iconNames';
+
+export enum ButtonSize {
+  Md = 'md',
```

Comment:
Enum names can use human readable string, rename it to Medium and Small. Keep the values in short form.